src/asset.ts

```diff
@@ -1,13 +1,14 @@
 import AinftBase from './ainftBase';
-import { AppCreditInfo, HttpMethod, NftContractBySymbol, NftToken, NftCollections } from './types';
+import { AppCreditInfo, HttpMethod, NftContractBySymbol, NftToken, NftCollections, NftMetadata } from './types';
 
 export default class Asset extends AinftBase {
   addNftSymbol(
     appId: string,
     chain: string,
-    contractAddress: string
+    contractAddress: string,
+    options?: Record<string, any>
```

Comment:
What is the `options`?

Reply:
Yes, you can use `setNftMetadata` after adding your contract to nft-server using `addNftSymbol`.
If you want to use set metadata api, you should add nft symbol with option: `{ isDynamic: true }`.
This option will prevent nft-pipeline from updating metadata itself.

Reply:
I got it, thank you for your kind explanation!